src/main/webapp/app/admin/statistics/statistics-graph.component.ts

```diff
@@ -0,0 +1,215 @@
+import { Component, Input, OnChanges, OnInit, SimpleChanges, ViewChild } from '@angular/core';
+import { StatisticsService } from 'app/admin/statistics/statistics.service';
+import { ChartDataSets, ChartOptions, ChartType } from 'chart.js';
+import { BaseChartDirective, Label } from 'ng2-charts';
+import { DataSet } from 'app/exercises/quiz/manage/statistics/quiz-statistic/quiz-statistic.component';
+import { TranslateService } from '@ngx-translate/core';
+import * as moment from 'moment';
+import { Graphs, SpanType } from 'app/entities/statistics.model';
+
+@Component({
+    selector: 'jhi-statistics-graph',
+    templateUrl: './statistics-graph.component.html',
+})
+export class StatisticsGraphComponent implements OnInit, OnChanges {
+    // Inputs
```

Comment:
You can remove that comment, that's why there's `@Input()` :smile:

Reply:
Done in [this](https://github.com/ls1intum/Artemis/pull/2511/commits/c16f4b7afac2d4b518fcd75ff6d28830a57ce8cf) commit